src/dmd/backend/dwarfdbginf.d

```diff
@@ -51,1532 +53,1514 @@ else
 
-    debugline = debugline_init;
+        /* ======================================== */
 
-    debug_line.buf.write(&debugline, debugline.sizeof);
+        debug_abbrev.initialize();
+        abbrevcode = 1;
 
-    // include_directories
-version (SCPP)
-    for (size_t i = 0; i < pathlist.length(); ++i)
-    {
-        debug_line.buf.writeString(pathlist[i]);
-        debug_line.buf.writeByte(0);
-    }
+        // Free only if starting another file. Waste of time otherwise.
+        if (abbrev_table)
+        {
+            AApair.destroy(abbrev_table);
+            abbrev_table = null;
+        }
 
-version (MARS) version (none)
-    for (int i = 0; i < global.params.imppath.dim; i++)
-    {
-        debug_line.buf.writeString((*global.params.imppath)[i]);
-        debug_line.buf.writeByte(0);
-    }
+        static immutable ubyte[21] abbrevHeader =
+        [
+            1,                      // abbreviation code
+            DW_TAG_compile_unit,
+            1,
+            DW_AT_producer,  DW_FORM_string,
+            DW_AT_language,  DW_FORM_data1,
+            DW_AT_name,      DW_FORM_string,
+            DW_AT_comp_dir,  DW_FORM_string,
+            DW_AT_low_pc,    DW_FORM_addr,
+            DW_AT_entry_pc,  DW_FORM_addr,
+            DW_AT_ranges,    DW_FORM_data4,
+            DW_AT_stmt_list, DW_FORM_data4,
+            0,               0,
+        ];
 
-    debug_line.buf.writeByte(0);              // terminated with 0 byte
+        debug_abbrev.buf.write(abbrevHeader.ptr,abbrevHeader.sizeof);
 
-    /* ======================================== */
+        /* ======================================== */
 
-    debug_abbrev.initialize();
-    abbrevcode = 1;
+        debug_info.initialize();
 
-    // Free only if starting another file. Waste of time otherwise.
-    if (abbrev_table)
-    {
-        AApair.destroy(abbrev_table);
-        abbrev_table = null;
-    }
+        debuginfo = debuginfo_init;
+        if (I64)
+            debuginfo.address_size = 8;
 
-    static immutable ubyte[21] abbrevHeader =
-    [
-        1,                      // abbreviation code
-        DW_TAG_compile_unit,
-        1,
-        DW_AT_producer,  DW_FORM_string,
-        DW_AT_language,  DW_FORM_data1,
-        DW_AT_name,      DW_FORM_string,
-        DW_AT_comp_dir,  DW_FORM_string,
-        DW_AT_low_pc,    DW_FORM_addr,
-        DW_AT_entry_pc,  DW_FORM_addr,
-        DW_AT_ranges,    DW_FORM_data4,
-        DW_AT_stmt_list, DW_FORM_data4,
-        0,               0,
-    ];
-
-    debug_abbrev.buf.write(abbrevHeader.ptr,abbrevHeader.sizeof);
-
-    /* ======================================== */
-
-    debug_info.initialize();
-
-    debuginfo = debuginfo_init;
-    if (I64)
-        debuginfo.address_size = 8;
-
-    // Workaround https://issues.dlang.org/show_bug.cgi?id=16563
-    // Struct alignment is ignored due to 2.072 regression.
-    static if (debuginfo.alignof == 1)
+        assert(debuginfo.alignof == 1);
```

Comment:
Same here regarding separate commit because of adjusted baseline compiler comment.